src/LocalInstaller.ts

```diff
@@ -144,6 +161,20 @@ export function resolve(packagesByTarget: ListByPackage) {
     return resolvedPackages;
 }
 
-function del(file: string): Promise<void> {
-    return fs.unlink(file);
+function del(dir: string) {
+    if (fs.existsSync(dir)) {
```

Comment:
Since we're now removing an entire directory, I think it's fine to include a dependency on `rimraf`. Its the defacto module to use for this purpose.